internal/pkg/manifest/doc.go

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package manifest provides functionality to create Manifest files.
+package manifest
```

Comment:
This is slick! We should populate this to the other pkgs.

Reply:
I was thinking we can do it everytime we touch a different package, instead of sending a massive refactor PR but I'm fine either way!

Reply:
oh no worries. I was thinking the same thing